test/PHPUnit/Controller/AbstractConsoleControllerTestCaseTest.php

```diff
@@ -8,9 +8,14 @@
  */
 namespace ZendTest\Test\PHPUnit\Controller;
 
+use PHPUnit\Framework\ExpectationFailedException;
 use Zend\Router\RouteMatch;
 use Zend\Test\PHPUnit\Controller\AbstractConsoleControllerTestCase;
 
+if (! class_exists(ExpectationFailedException::class)) {
+    class_alias(\PHPUnit_Framework_ExpectationFailedException::class, ExpectationFailedException::class);
+}
```

Comment:
Ugh, wish I'd had a chance to review before merge.

All these `class_alias` calls should likely be in a single file that is added via an `autoload_dev` section of the `composer.json`. That way the aliases are created when needed, and outside the class file declarations themselves.